packages/sources/fireblocks/src/endpoint/reserve.ts

```diff
@@ -0,0 +1,35 @@
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import {
+  PoRProviderEndpoint,
+  PoRProviderResponse,
+} from '@chainlink/external-adapter-framework/adapter/por'
+import { config } from '../config'
+import { httpTransport } from '../transport/reserve'
+
+export const inputParameters = new InputParameters(
+  {
+    accountName: {
+      type: 'string',
+      description: 'The name of the account to retrieve balances for',
+      default: 'USDFB',
```

Comment:
Do we want to have a default value here? Could accidentally mask if a job has incorrect input params (as the EA will return a value rather than error)